### DISCUSSION TITLE
chown/chmod .dss file

### DISCUSSION THREAD

**Discussion by mdlaat** [007revad/Synology_Config_Backup#23] [Ideas]
Thank you for this nice script!
Since it is being run as root, the resulting .dss file is owned by root.
It would be nice to have an option in the script to chown/chmod the .dss file after it is created.

**Comment by 007revad** [comment]
Interesting. I never noticed this before.

Each of my 3 Synology NAS export the dss file locally and then copy it to the other 2 NAS. 
- The local dss exports are root:root 0000 via SSH (but via File Station all users have read/write allow).
- The dss files copied to the other NASes have Dave:users 0700.

I'm thinking the easiest solution would be to chown admin:administrators and chmod 0770 so any user with admin privileges can access the file.

EDIT chown admin:administrators also does chmod 0770

**Reply by 007revad** [reply]
I've fixed it in v1.1.16 
https://github.com/007revad/Synology_Config_Backup/releases
